### PR TITLE
[BPK-941] Tokenize font families for RN

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: packages/bpk-tokens/tokens/android/base.raw.json [hunk 1]
```diff
@@ -854,6 +860,24 @@
       },
       "name": "SPACING_XXL"
     },
+    "FONT_FAMILY": {
+      "value": "\"sans-serif\"",
```
Comment (k0nserv) [inline packages/bpk-tokens/tokens/android/base.raw.json]: This is a bit of a hack as this just gets serialized in Javascript we need the quotes to make sure it ends up as a stirng in JS

File: native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ const styles = StyleSheet.create({
     fontSize: textXsFontSize,
     fontWeight: textXsFontWeight,
     // TODO Replace with fontFamilyXs token once available
-    fontFamily: Platform.OS === 'android' ? 'sans-serif' : 'System',
+    fontFamily: Platform.OS === 'android' ? tokens.fontFamily : tokens.fontFamily,
```
Comment (matthewdavidson) [inline native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js]: This seems like superfluous ternary :P

File: packages/bpk-tokens/src/react-native-output-test.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ import _ from 'lodash';
 import androidTokens from '../tokens/android/base.raw.json';
 import iosTokens from '../tokens/ios/base.raw.json';
 
-const ANDROID_ONLY_PROPS = [];
+const ANDROID_ONLY_PROPS = ['FONT_FAMILY_MEDIUM'];
```
Comment (matthewdavidson) [inline packages/bpk-tokens/src/react-native-output-test.js]: What is better... 

**Baking conditions into tokens:**
_ios:_
```
FONT_FAMILY: 'System'
FONT_FAMILY_MEDIUM: 'System'
```
_android:_
```
FONT_FAMILY: 'sans-serif'
FONT_FAMILY_MEDIUM: 'sans-serif-medium'
```
_component:_
```
style: {
  fontFamily: emphasized ? TOKENS.FONT_FAMILY : TOKENS.FONT_FAMILY_MEDIUM
}
```

Or, **having conditional tokens and adding extra conditions into components:**
_component:_
```
style: {
  fontFamily: Platform.OS === 'android' ? (emphasized ? TOKENS.FONT_FAMILY : TOKENS.FONT_FAMILY_MEDIUM)  : TOKENS.FONT_FAMILY
}
```

File: native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js (hunk 2 above)
Comment (k0nserv) [reply native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js]: Haha good catch :joy:

File: packages/bpk-tokens/tokens/android/base.raw.json (hunk 1 above)
Comment (georgegillams) [reply packages/bpk-tokens/tokens/android/base.raw.json]: I think when I was doing this originally I used `"'sans-serif'"` to achieve the same thing

File: packages/bpk-tokens/src/react-native-output-test.js (hunk 3 above)
Comment (k0nserv) [reply packages/bpk-tokens/src/react-native-output-test.js]: I like making it explict as this is a techincal implementation detail of Android and thus the code should relfect that we have a special case here.

File: native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js [hunk 4]
```diff
@@ -77,7 +77,7 @@ const styles = StyleSheet.create({
     color: colorGray700,
     fontWeight: textLgFontWeight,
     // TODO Replace with fontFamilyLg token once available
```
Comment (matteo-hertel) [inline native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js]: is the todo still relevant?

File: packages/bpk-tokens/src/react-native-output-test.js (hunk 3 above)
Comment (matteo-hertel) [reply packages/bpk-tokens/src/react-native-output-test.js]: react native utils? :trollface:

File: native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js (hunk 4 above)
Comment (k0nserv) [reply native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js]: Nope, I thought I removed that

Comment (matteo-hertel) [reply native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js]: never mind, I submitted the review while you were pushing 😅

**Pull request merged**